Assets/MRTK/Providers/XRSDK/Microsoft.MixedReality.Toolkit.Providers.XRSDK.asmdef

```diff
@@ -1,9 +1,10 @@
 {
     "name": "Microsoft.MixedReality.Toolkit.Providers.XRSDK",
     "references": [
-        "Microsoft.MixedReality.Toolkit"
+        "Microsoft.MixedReality.Toolkit",
+        "Unity.XR.Management",
```

Comment:
I don't think we want to check these changes in. It's updated automatically when the project is loaded in 2019, but left un-checked-in to preserve 2018 support.

Reply:
Oh nooooo I didn't mean to include this one